### PR TITLE
WRP-17253: Updated node modules major version

### PR DESCRIPTION
**Pull request by ion-andrusciac-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updated node modules major version

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Updated (there were no breaking changes affecting our code):
`@enact/dev-utils`
https://github.com/enactjs/dev-utils/releases

`postcss-loader`
 https://github.com/webpack-contrib/postcss-loader/releases

`postcss-preset-env`
https://github.com/csstools/postcss-plugins/blob/main/plugin-packs/postcss-preset-env/CHANGELOG.md
[Migration Guide](https://github.com/csstools/postcss-plugins/wiki/PostCSS-Preset-Env-8/#postcss-preset-env-breaking-changes) - there are no breaking changes affecting Enact

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
[@enact/dev-utils](https://github.com/enactjs/dev-utils/releases#:~:text=Updated%20the%20minimum%20version%20of%20Node%20to%2014.15.0) and [postcss-loader](https://github.com/webpack-contrib/postcss-loader/blob/master/CHANGELOG.md#:~:text=minimum%20supported%20Node.js%20version%20is%2014.15.0), the minimum required node version is v14.15.0. [postcss-preset-env](https://github.com/csstools/postcss-plugins/blob/main/plugin-packs/postcss-preset-env/CHANGELOG.md#:~:text=for%20usage%20details.-,Updated%20Support%20for%20Node%20v14%2B%20(major).,-Fixed%3A%20all) supports Node v14+

storybook dependencies will be handled in a separate user story

### Links
[//]: # (Related issues, references)
WRP-17253

### Comments

Enact-DCO-1.0-Signed-off-by: Ion Andrusciac ion.andrusciac@lgepartner.com

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]

**Comment by hong6316** [comment]
I don't think the node version required is currently correct.
In the case of dev-utils, the node version has been changed as follows.
"node": "^14.15.0 || ^16.10.0 || >=18.0.0.
The storybook-utils should also be changed to the correct node version by checking the breaking changes of each major updated module

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ## unreleased
 
+* Updated `@enact/dev-utils` version to `^6.0.0`.
```
Comment (hong6316) [inline CHANGELOG.md:3]: Please modify your changelog to be brief like the below link.
https://github.com/enactjs/cli/pull/321/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed

**Comment by ion-andrusciac-lgp** [comment]
> 1. I don't think the node version required is currently correct.
>    In the case of dev-utils, the node version has been changed as follows.
>    `"node": "^14.15.0 || ^16.10.0 || >=18.0.0`.
>    The storybook-utils should also be changed to the correct node version by checking the breaking changes of each major updated module
> 2. postcss-preset-env has a migration guide: https://github.com/csstools/postcss-plugins/wiki/PostCSS-Preset-Env-8
>    Did you check that?

1.  I updated the node version in package.json based on all restrictions from the three updated dependencies, thank you for noticing.
2. I checked Migration Guide and there are no Breaking Changes affecting the storybook-utils

**Review by hong6316 — APPROVED** [review]
👍

**Pull request merged**